server/etcdmain/grpc_proxy.go

```diff
@@ -130,6 +131,7 @@ func newGRPCProxyStartCommand() *cobra.Command {
 	cmd.Flags().StringVar(&grpcProxyMetricsListenAddr, "metrics-addr", "", "listen for endpoint /metrics requests on an additional interface")
 	cmd.Flags().BoolVar(&grpcProxyInsecureDiscovery, "insecure-discovery", false, "accept insecure SRV records")
 	cmd.Flags().StringSliceVar(&grpcProxyEndpoints, "endpoints", []string{"127.0.0.1:2379"}, "comma separated etcd cluster endpoints")
+	cmd.Flags().DurationVar(&grpcProxyClientAutoSyncInterval, "client-auto-sync-interval", 0, "etcd endpoints auto sync interval (disabled by default)")
```

Comment:
rename to `endpoints_auto_sync_interval`?

Reply:
Yep, looks better. Changed.